### ISSUE TITLE
Card's name property drops last character

### ISSUE THREAD

**Issue by ashfurrow** [CardFlight/cardflight-v3-ios#2]
Hi there! I'm tokenizing cards I read and grabbing the name from the card, but it's dropping the final character. For example:

```swift
func readerCardResponse(_ card: CFTCard?, withError error: Error?) {
    card.tokenizeCard(success: { in
        print(card.name) // prints ASHLEY FURRO (missing a W from the end)
    }
}
```

I was experiencing this on version 3.2.1 and tried upgrading to 3.6 but it's still not working. Any tips?

**Comment by ashfurrow** [comment]
Just tried one of the newer readers, and it works as expected. The older readers we're using (`CFTReaderType_SHUTTLE`) presents the error though.

**Comment by ashfurrow** [comment]
Hi there, just following up on this. Is there any more info I can supply to help?

**Comment by hokiewalrus** [comment]
We are tracking this issue and it will be fixed in the next release

**Comment by ashfurrow** [comment]
Awesome, thanks for the heads up 👍

**Comment by hokiewalrus** [comment]
This is now fixed in v3.6.1
